extensions/proposals/ARB_texture_storage_multisample/extension.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- vi:set sw=2 ts=4: -->
+<?xml-stylesheet href="../../extension.xsl" type="text/xsl"?>
+<proposal href="proposals/ARB_texture_storage_multisample/">
+  <name>ARB_texture_storage_multisample</name>
+
+  <contact> <a href="https://www.khronos.org/webgl/public-mailing-list/">WebGL
+  working group</a> (public_webgl 'at' khronos.org) </contact>
+
+  <contributors>
+    <contributor>Jeff Gilbert, Mozilla</contributor>
+  </contributors>
+
+  <number>XYZ</number>
+
+  <depends>
+    <api version="2.0"/>
+  </depends>
+
+  <overview>
+    <mirrors href="https://www.khronos.org/registry/OpenGL/extensions/ARB/ARB_texture_storage_multisample.txt"
+             name="ARB_texture_storage_multisample">
```

Comment:
This is problematic because this is an OpenGL extension, not an OpenGL ES extension. Previously when subsetting OpenGL functionality like https://www.khronos.org/registry/webgl/extensions/WEBGL_depth_texture/ and https://www.khronos.org/registry/webgl/extensions/ANGLE_instanced_arrays/ did, first an OpenGL ES extension was written (under the ANGLE_ prefix, for example), and then the WebGL extension wrapped it.

Is this intended to expose ES 3.1's multisampled texture support? If so, can this explicitly be stated?

If you really want to reference the ARB_ version, can you add some addendum that references to the 1D and rectangular texture types are dropped (because they don't exist in ES 3.0)? Are there other references to OpenGL-specific features or functionality that don't exist in ES?